src/plugins/intel_gna/src/backend/dnn_types.cpp

```diff
@@ -30,6 +30,7 @@ const char* intel_dnn_operation_name[kDnnNumOp] = {"kDnnNullOp",
                                                    "kDnnConvolutional2dOp",
                                                    "kDnnPiecewiselinearOp",
                                                    "kDnnMaxPoolOp",
+                                                   "kDnnMaxPool2dOp",
```

Comment:
Did you extend also the types for model dump?

Reply:
change removed